dev-manuals/test-plans/minigames/regex-game-frontend/plans/u-regex-game-1-game-start.md

```diff
@@ -0,0 +1,35 @@
+# Regex Game: Game Start (`u.regex-game-1`)
+
+Version: V1.0, 09.08.2022, initial test \
+Author: Leon Hofmeister
+
+## Description
+
+Tests that the game can be started.
+
+## Precondition
+
+The tester has a copy of the game.
+
+## Postcondition
+
+The game was started.
+
+## Typical procedure
+
+1. Press start
+2. The game starts
+3. The first regex is loaded (from the backend, once available)
```

Comment:
just write `from the backend`

Reply:
There is no backend yet.
I'd like not to merge tests that must always fail because the functionality isn't present yet.